src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -419,19 +429,45 @@ public boolean getAutoRefresh() {
     }
 
     /**
-     * Gets the maximum number of times an API request will be tried when an error occurs.
+     * Sets the total maximum number of times an API request will be tried when error responses
+     * are received.
      * @return the maximum number of request attempts.
+     * @deprecated getMaxRetryAttempts is preferred because it more clearly gets the number
+     * of times a request should be retried after an error response is received.
      */
+    @Deprecated
     public int getMaxRequestAttempts() {
-        return this.maxRequestAttempts;
+        return this.maxRetryAttempts + 1;
```

Comment:
```suggestion
        return this.maxRetryAttempts;
```

Reply:
The + 1 should be removed because when our retry logic calls this method, it should return the number of retries. It will add 1 to the number of retries when it executes.

Reply:
This method is deprecated, so the SDK doesn't call it anymore.  It's just preserved for client apps that may have been calling it.

Reply:
This method is deprecated, so the SDK doesn't call it anymore. It's just preserved for client apps that may have been calling it.